docker/Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM ruby:2.7.5
+FROM ruby:3.2.0
```

Comment:
This is an obsolete patch version — 3.2.2. is current.

Reply:
ok, will update